cmd/argoexec/commands/resource.go

```diff
@@ -34,6 +34,7 @@ func NewResourceCommand() *cobra.Command {
 func execResource(ctx context.Context, action string) error {
 	wfExecutor := initExecutor()
 	defer wfExecutor.HandleError(ctx)
+	defer wfExecutor.FinalizeOutput(ctx)
```

Comment:
good catch - I just saw in the code that there's no wait container for "argoexec resource", huh?

Reply:
Yes, it didn't have wait container.

Reply:
We need to mark the tasksult to true for these templates that do not have a wait container but have outputs. https://github.com/argoproj/argo-workflows/pull/12544

Reply:
thanks. yeah, I added a couple of comments over there that would be related here as well